### DISCUSSION TITLE
Can we put non-tech related topics here instead of issues

### DISCUSSION THREAD

**Discussion by 0xmerkle** [HumanAIGC/OutfitAnyone#14] [General]
I like that there is a lot of discussion around this project, however, I think some of the discussions I found on the issues tab may be more suited for the discussions tab. Excited to chat about the possibilities of this here with y'all! :)

**Comment by danatodd** [comment]
Good point, and I wish I'd seen your post prior to hitting "reply" on the email thread I got which posted my response in Issues.
